### PR TITLE
Added result page

### PR DESCRIPTION
**Pull request by Amandeep-sinha**
please review my PR for Result page

image

check live at :- https://trusting-jackson-88f49a.netlify.app/result.html

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,170 @@
+:root {
+    --backcolor: #222;
```
Comment (Sen-442b) [inline style.css:2]: Suggestion

- You can use a hyphen to separate distinct words

File: style.css [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+    width: fit-content;
+}
+
+h1 {
+    text-align: center;
+    font-size: 3rem;
+    margin-bottom: .5rem;
+}
+
+ul {
```
Comment (preethasri) [inline style.css:96]: try to use class for this and all the tag that you are directly selecting use classname for that also .rest all looks good👍

**Review by preethasri — APPROVED** [review]

File: style.css [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+    margin: 0;
+    background-color: var(--backcolor);
+    color: var(--primary);
+}
+
+.trivia-header-div {
+    background-image: url('https://bit.ly/3p1IZrt');
+    height: 600px;
+    width: 100%;
+    padding: 0%;
```
Comment (Sen-442b) [inline style.css:23]: You don't need to add the percent unit if the value is zero

File: style.css [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+
+.next-div button {
+    margin-right: 8rem;
+}
+
+.score-div {
+    font-size: 2rem;
+    text-align: center;
+    color: yellow;
+    border: solid green 4px;
```
Comment (Sen-442b) [inline style.css:149]: (nitpick) 
You can follow a consistent approach while giving shorthands  eg:-` border:4px solid green`

**Review by Sen-442b — APPROVED** [review]
Other things look good. 🔥

**Pull request merged**